alacritty.yml

```diff
@@ -230,6 +230,8 @@ visual_bell:
   duration: 0
 
 # Background opacity
+# The value can be between 0 and 1.
+# Values less than 1 will add some transparency on the window.
```

Comment:
This should follow the layout of the rest of the configuration file, so between the description and the title, there should be one empty line.

The wording of the description could also be improved a bit. I'd propose something like this:
```yml
# Background opacity
#
# Window opacity as a floating point number from `0.0` to `1.0`.
# The value `0.0` is completely transparent, and `1.0` is opaque.
```